source/extensions/resource_monitors/fixed_heap/fixed_heap_monitor.cc

```diff
@@ -22,10 +25,25 @@ FixedHeapMonitor::FixedHeapMonitor(
 }
 
 void FixedHeapMonitor::updateResourceUsage(Server::ResourceUpdateCallbacks& callbacks) {
-  const size_t physical = stats_->reservedHeapBytes();
-  const size_t unmapped = stats_->unmappedHeapBytes();
-  ASSERT(physical >= unmapped);
-  const size_t used = physical - unmapped;
+
+  size_t used;
+
+  if(!Runtime::runtimeFeatureEnabled("envoy.reloadable_features.do_not_count_mapped_pages_as_free"))
+  {
```

Comment:
Run format (braces not on new line)

Reply:
Seems like you still need to run format, CI failing for format

Reply:
Done